versions/schematic-2.md

```diff
@@ -0,0 +1,235 @@
+**This specification is under development**
+# Sponge Schematic Specification
+
+**Version 2**
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](http://www.ietf.org/rfc/rfc2119.txt).
+
+## Introduction
+
+This specification defines a format which describes an area of a [Minecraft](http://minecraft.net) world for the purpose of serialization to disk and deserialization from disk. It is designed in order to allow maximum cross-compatibility between platforms, versions, and various states of modification.
+
+## Revision History
+
+Version | Date | Notes
+---|:---:|---
+1 | 2016-08-23 | Initial Version
+2 | 2018-08-19 | More features, easier handling, see [Changelog](#changelog)
+
+## Specification
+
+### Format
+
+All field names in the specification are **case sensitive**.
+
+#### <a name="formFile"></a>File
+
+The structure described by this specification is persisted to disk using the [Named Binary Tag](http://minecraft.gamepedia.com/NBT_format) (NBT) format (also see [here](https://wiki.vg/NBT)). Before writing to disk the NBT data must be compressed using the [GZip](https://www.gnu.org/software/gzip/) data compression algorithm. The highly recommended file extension for files using this specification is `.schem` or `.spongeschem` (chosen so as to not conflict with the legacy `.schematic` format allowing easy distinction between the two).
+
+#### <a name="formId"></a>Identification string
+
+Resources identified by a resource location have an identifier string which is made up of two sections, the domain and the name, written as `domain:name`. If the domain is not specified then it is assumed to be `minecraft`. For example `planks` would identify the same resource as `minecraft:planks` as the `minecraft:` prefix is implicit.
+
+### Schema
+
+#### <a name="schematicObject"></a>Schematic Object
+
+This is the nameless root object of the schematic. It shall be a [NBT](#formFile) tag compound.
+
+##### Fields
+
+Field Name | Type | Description
+---|:---:|---
+<a name="schematicVersion"></a>Version | `NBT int` | **Required.** Specified the format version being used. It may be used to provide validation and auto-conversion from older versions. The current version is `2`. This shall be saved as [NBT](#formFile) tag int.
+<a name="schematicContentVersion"></a>ContentVersion | `NBT int` | **Required.** A version identifier for the contents. Representing the minecraft version, when serialized. Used for providing backwards compatibility. [Values are specified by Mojang](https://minecraft-de.gamepedia.com/Versionen#Version-IDs). Mojangs data converter for older versions will need this value for proper conversion. This shall be saved as [NBT](#formFile) tag int.
```

Comment:
I totally agree on "DataVersion".
Yea, i could add an example.
I already linked the german MC Wiki with these values (https://minecraft-de.gamepedia.com/Versionen#Version-IDs), because i didn't find this Wiki chapter in the english wiki yet... did you?

One question to your replacement. "This can be used to", should we replace it by "This SHALL be used to"?